core/space/services/services_fs.go

```diff
@@ -41,7 +34,13 @@ func (s *Space) listDirAtPath(
 			continue
 		}
 
-		relPath := relPathRegex.FindStringSubmatch(item.Path)[2]
+		paths := relPathRegex.FindStringSubmatch(item.Path)
```

Comment:
this is a potential fix @dmerrill6 but not sure if we need to come back to this, for now it seems to work